.github/workflows/ci.yml

```diff
@@ -57,6 +57,7 @@ jobs:
         with:
           ruby-version: ${{ matrix.ruby }}
           bundler-cache: true
+          bundler: latest
```

Comment:
What happens without this?

Reply:
This was just testing on the jruby issue.  I've removed now.